webdriver/tests/bidi/browsing_context/capture_screenshot/invalid.py

```diff
@@ -123,28 +122,28 @@ async def test_params_clip_viewport_width_invalid_type(
 
 
 @pytest.mark.parametrize("value", [None, False, "foo", {}, []])
-async def test_params_clip_viewport_height_invalid_type(
-    bidi_session, top_context, value
-):
+async def test_params_clip_box_height_invalid_type(bidi_session, top_context,
+                                                   value):
     with pytest.raises(error.InvalidArgumentException):
         await bidi_session.browsing_context.capture_screenshot(
             context=top_context["context"],
             clip=BoxOptions(x=0, y=0, width=0, height=value),
         )
 
 
-async def test_params_clip_viewport_dimensions_invalid_value(bidi_session, top_context):
+async def test_params_clip_box_dimensions_invalid_value(
+        bidi_session, top_context):
     with pytest.raises(error.UnableToCaptureScreenException):
         await bidi_session.browsing_context.capture_screenshot(
             context=top_context["context"],
             clip=BoxOptions(x=0, y=0, width=0, height=0),
         )
 
 
-async def test_params_clip_viewport_outside_of_window_viewport(
-    bidi_session, top_context
-):
-    viewport_dimensions = await get_viewport_dimensions(bidi_session, top_context)
+async def test_params_clip_box_outside_of_window_viewport(
```

Comment:
I would suggest to move the tests that do not test for `invalid argument` to the corresponding test module. In this case it looks like to be related to `clip.py`. This also applies to the next one.